modules/rosa-create-dedicated-cluster-admins.adoc

```diff
@@ -4,8 +4,8 @@
 
 
 [id="rosa-create-dedicated-cluster-admins.adoc"]
-= Granting dedicated-admin and cluster-admin access
-As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. Additionally, only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges.
+= Granting dedicated-admin access
+Only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges. As a best practice, grant `dedicated-admin` access to most of your administrators.
```

Comment:
Same comment as above about not starting sentence with lowercase.

Reply:
Done.